### DISCUSSION TITLE
希望其他商店里的功勋商店也能增加一个单独开关

### DISCUSSION THREAD

**Discussion by ChaosNiku** [LmeSzinc/AzurLaneAutoScript#2054] [Ideas]
功勋商店现在默认会兑换紫菜和蓝材还有图纸，没法直接手动取消，比较不方便，希望能在改造部件和强化部件兑换选项上加个不兑换或者给退役商店单独加个开关，这样开启其他商店的时候就可以不影响大舰队等的兑换了

**Comment by LmeSzinc** [comment]
这里有过滤器
![image](https://user-images.githubusercontent.com/37934724/210138406-c68df2ad-9881-4c0f-a7a7-14ca43960c34.png)
